app/forms/declaration/ContainerFirst.scala

```diff
@@ -61,8 +61,7 @@ object ContainerFirst extends DeclarationPage {
       yes,
       text()
         .verifying("declaration.transportInformation.containerId.empty", nonEmpty)
-        .verifying("declaration.transportInformation.containerId.error.alphanumeric", isAlphanumeric)
-        .verifying("declaration.transportInformation.containerId.error.length", noLongerThan(maxContainerIdLength))
+        .verifying("declaration.transportInformation.containerId.error.invalid", isAlphanumeric and noLongerThan(maxContainerIdLength))
```

Comment:
Here probably `isEmpty` is missing. In case that the input will be empty we can have two errors (Not sure that `isAlphanumeric` return `true` or `false` for empty input)

Reply:
Seems to work without it.  But I'll add it to be consistent.  (For both comments)